src/api/Reducers/propertyTree.js

```diff
@@ -68,13 +68,41 @@ const propertyOwners = (state = {}, action = {}) => {
           subowners: owner.subowners,
           tags: owner.tags || []
         };
+        
+        // Ensure the parents of the uri have the links to the new entry
+        // Get parent uri
+        var periodPos = owner.uri.lastIndexOf('.');
```

Comment:
```suggestion
        let periodPos = owner.uri.lastIndexOf('.');
```

Reply:
could be `const`? (same for the comments below)